### PR TITLE
Documentation for "ask the community" feature

### PR DESCRIPTION
**Pull request by jasonblais**
This is part 2 to previous documentation updates at https://github.com/mattermost/docs/pull/3766

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -1310,6 +1310,16 @@ Custom text will be shown below custom brand image on left side of server login
 | This feature's ``config.json`` setting is ``"CustomBrandText": ""`` with string input.                                                                               |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Ask Community Link
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When true, "Ask the community" link appears on the Mattermost user interface and Main Menu, which allows users to join the Mattermost Community to ask questions and help others troubleshoot issues.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:1315]:
```suggestion
**True**: **Ask the community** link is visible on the Mattermost user interface and Main Menu. When selected, users can join the Mattermost Community to ask questions and help others troubleshoot issues.
```

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -1310,6 +1310,16 @@ Custom text will be shown below custom brand image on left side of server login
...
+
+When false, the link is hidden from users.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:1317]:
```suggestion
**False**: The link is not visible to users.
```

**Comment by justinegeffen** [comment]
Thanks, @jasonblais. Is there a default URL provided when the setting is ``true``? Or does that need to be added?

**Comment by jasonblais** [comment]
@justinegeffen Yes, I added the URL for reference, good question

**Comment by justinegeffen** [comment]
Another question, sorry to be a pain, but does the user need to sign up or can they use existing credentials? And does that open in a browser?

**Comment by jasonblais** [comment]
The full experience is:

1. User clicks on "Ask the community" item
2. User is taken to https://mattermost.com/pl/default-ask-mattermost-community/, which is an intermediate landing page providing additional context on benefits and how to get started. The page opens in their default browser, and is not available on mobile app.
3. The user is then taken to https://community.mattermost.com/?redirect_to=%2Fcore%2Fchannels%2Fpeer-to-peer-help to create an account in the community server. After account creation, they'll land directly in the peer-to-peer help channel.

**Comment by justinegeffen** [comment]
Thanks, @jasonblais. I was thinking we should add that to the doc, but it's really straightforward. Wondering though if we should mention that it's not available on the mobile app?

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -1310,6 +1310,16 @@ Custom text will be shown below custom brand image on left side of server login
 | This feature's ``config.json`` setting is ``"CustomBrandText": ""`` with string input.                                                                               |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Ask Community Link
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**True**: **Ask the community** link is visible in the Mattermost channel header, under the **Help** menu. When clicked, users are redirected to https://mattermost.com/pl/default-ask-mattermost-community/, where they can join the Mattermost Community to ask questions and help others troubleshoot issues.
```
Comment (jasonblais) [inline source/administration/config-settings.rst:1315]:
```suggestion
**True**: **Ask the community** link is visible in the Mattermost channel header, under the **Help** menu. When clicked, users are redirected to https://mattermost.com/pl/default-ask-mattermost-community/, where they can join the Mattermost Community to ask questions and help others troubleshoot issues. This option is not available on the mobile apps.
```

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -1310,6 +1310,16 @@ Custom text will be shown below custom brand image on left side of server login
...
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Ask Community Link
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**True**: **Ask the community** link is visible in the Mattermost channel header, under the **Help** menu. When clicked, users are redirected to https://mattermost.com/pl/default-ask-mattermost-community/, where they can join the Mattermost Community to ask questions and help others troubleshoot issues. This option is not available on the mobile apps.
+
+**False**: The link is not visible to users.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"enable_ask_community_link": ""`` with options ``true`` and ``false``.                                                   |
```
Comment (esethna) [inline source/administration/config-settings.rst:1320]: value is true by default?

**Comment by jasonblais** [comment]
Thanks! Added default value

**Pull request merged**